ui/src/overview.tsx

```diff
@@ -91,17 +90,31 @@ function AddCriteriaButton(props: { group: string | GroupKind }) {
     [props.group]
   );
 
-  const items = getCriteriaTitles();
+  // TODO(tjennison): Fetch configs from the backend.
+  const configs = [
+    {
+      type: "concept",
```

Comment:
Does this need some kind of linkage to the `condition_occurrence` table as well? Or not yet?

Reply:
It will at some point to handle the modifiers. We're still going to need to planning to figure out how to handle those connections generically.